mobile-app/lib/features/main/screens/send/send_screen.dart

```diff
@@ -219,14 +217,17 @@ class SendScreenState extends State<SendScreen> {
       setState(() {
         _networkFee = estimatedFee;
         _isFetchingFee = false;
-        _hasAmountError = (_amount + _networkFee) > _maxBalance;
+        _hasAmountError = (_amount + estimatedFee) > _maxBalance;
       });
+
+      return estimatedFee;
     } catch (e) {
       print('Error fetching network fee: $e');
       setState(() {
         _isFetchingFee = false;
```

Comment:
Apart from it being changed, kind of looks wrong anyway - we just ignore network fee here... hmm.